### PR TITLE
Adjust project for 07/10 changes, proto files update

### PR DESCRIPTION
**Pull request by ice09**

### CODE REVIEW

File: src/main/proto/mod.rs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
```
Comment (ketola) [inline src/main/proto/mod.rs]: Is this file really necessary, it's not a proto file?

File: src/main/java/dev/jlibra/example/GetAccountStateExample.java [hunk 2]
```diff
@@ -23,7 +23,7 @@
 
 public class GetAccountStateExample {
 
-    public static void main(String[] args) throws IOException {
+    public static void main(String[] args) throws IOException, InterruptedException {
```
Comment (ketola) [inline src/main/java/dev/jlibra/example/GetAccountStateExample.java]: This one does not have the Thread.sleep added, should it have? Or why was he exception added?

File: src/main/proto/mod.rs (hunk 1 above)
Comment (ice09) [reply src/main/proto/mod.rs]: Yes, it was added by libra but is not necessary for generation.

File: src/main/java/dev/jlibra/example/GetAccountStateExample.java (hunk 2 above)
Comment (ice09) [reply src/main/java/dev/jlibra/example/GetAccountStateExample.java]: Yes, removed it.

File: src/main/java/dev/jlibra/example/TransferExample.java [hunk 3]
```diff
@@ -88,6 +88,8 @@ public static void main(String[] args) throws Exception {
 
         SubmitTransactionResponse response = stub.submitTransaction(submitTransactionRequest);
 
+        Thread.sleep(2000); // add sleep to prevent premature closing of channel
```
Comment (hczerpak) [inline src/main/java/dev/jlibra/example/TransferExample.java]: I saw issue submitted earlier related to this. I'm quite surprised this happens because `submitTransaction` is a blocking call.

Comment (ketola) [reply src/main/java/dev/jlibra/example/TransferExample.java]: Yes, what's even more weird is that the response has been received also at this point. Would be interesting to know more about this, but I have not witnessed the issue. Where did you see this problem mentioned?

Comment (ice09) [reply src/main/java/dev/jlibra/example/TransferExample.java]: I had this problem before (could reproduce with positive and negative tests), not always but quite often. @Ade92 had this same problem and Thread.sleep solved it for him as well: https://github.com/ketola/jlibra/issues/10

I had similar problems already with gRPC, so that's why I tried this. Usually this is never recognized as applications rarely exit. But I don't know how this can happen, just assume a Threading / not joining Threads before exiting issue.
